app/javascript/src/PaymentGateways/stripe/components/StripeCardForm.jsx

```diff
@@ -132,6 +135,13 @@ const StripeCardForm = ({ setupIntentSecret, billingAddressDetails, successUrl,
             className="col-md-12"
             onChange={validateCardElement}
           />
+          <input
+            type="text"
+            id="cardholder-name"
+            placeholder="Cardholder's name (optional)"
+            value={cardName}
+            onChange={onChange(setCardName)}
```

Comment:
Do we need this `onChange` function here? If so, I suggest we have another name for it, so we can avoid the name duplication.